sdk/src/types/block/output/native_token.rs

```diff
@@ -20,6 +20,7 @@ use crate::types::block::{output::TokenId, Error};
 #[packable(unpack_error = Error)]
 pub struct NativeToken {
     // Identifier of the native token.
+    #[serde(rename = "id")]
```

Comment:
cant we migrate this to keep it token_id? rename should be avoided as much as possible id say

Reply:
It's only id in the node API

Reply:
Yeah it's the protocol

Reply:
And there is no rename, it was already `id`, I just move the serde to the main type instead of the Dto

Reply:
wellshould we jsut call it id as well then?

Reply:
To be a bit precise, the protocol TIP call it token_id and the API call it id, which is exactly what we have atm, so I'd rather commit to the TIPs 🤷🏻‍♂️

Reply:
I know it's weird / not ideal but that's how it is